hack/make/prerequisites.mk

```diff
@@ -44,7 +44,7 @@ SETUP_ENVTEST ?= $(LOCALBIN)/setup-envtest
 PYTHON ?= $(LOCALBIN)/.venv/bin/python3
 MARKDOWNLINT ?= $(LOCALBIN_NPM)/markdownlint
 
-ENVTEST_K8S_VERSION = $(shell go list -m -f "{{ .Version }}" k8s.io/api | sed 's/v0/1/' )
+ENVTEST_K8S_VERSION = $(shell go list -m -f "{{ .Version }}" k8s.io/api | awk -F'[v.]' '{printf "1.%d", $$3}')
```

Comment:
based on kubebuilder, may be need to update it again https://github.com/kubernetes-sigs/kubebuilder/blob/b5a66b092494b607202870792cc58c875cdb450e/pkg/plugins/golang/v4/scaffolds/internal/templates/makefile.go#L273-L275